eng/Build.ps1

```diff
@@ -292,13 +288,11 @@ function BuildSolution([string] $solutionName, $nopack) {
     # Do not set the property to true explicitly, since that would override value projects might set.
     $suppressExtensionDeployment = if (!$deployExtensions) { "/p:DeployExtension=false" } else { "" }
 
-    $sourceBuildArgs = if ($sourceBuild) { "/p:DotNetBuildSourceOnly=true /p:DotNetBuildRepo=true" } else { "" }
-
     $BUILDING_USING_DOTNET_ORIG = $env:BUILDING_USING_DOTNET
 
     $env:BUILDING_USING_DOTNET="false"
 
-    $pack = if ($nopack -eq $False) {""} else {$pack}
+    $pack = if ($packSolution -eq $False) {""} else {$pack}
```

Comment:
Since the semantics got inverted (from noPack to packSolution), this also is logically inverted.. ?